src/ape/api/providers.py

```diff
@@ -81,6 +83,12 @@ def validate_hexbytes(cls, value):
             raise ValueError(f"Hash `{value}` is not a valid Hexbyte.")
         return value
 
+    @cached_property
+    def transactions(self) -> List[TransactionAPI]:
+        query = BlockTransactionQuery(block_id=self.hash)
+        df_txns = self.query_manager.query(query)
+        return list(map(lambda x: TransactionAPI(x), df_txns.values.tolist()))
```

Comment:
Functionaliity the same
```suggestion
        return list(map(TransactionAPI, df_txns.values.tolist()))
```